nix/docker.nix

```diff
@@ -50,6 +50,10 @@ let
     experimental-features = nix-command flakes
   '';
 
+  etcReleaseName = writeTextDir "etc/coderniximage-release" ''
+    0.0.0
```

Comment:
nit: wonder if it would make sense to have some kind of versioning here?

Reply:
I agree. It would make sense to get some versioning here.

Right now, we're not versioning the dogfood images, so this file is mostly a stub/placeholder for the system info go library not to fail.

Reply:
That or git SHA would probably be fine!